shared/actions/tracker.js

```diff
@@ -636,3 +650,9 @@ export function pendingIdentify (username: string, pending: boolean): PendingIde
     payload: {username, pending},
   }
 }
+
+export function openProofUrl (proof: Proof): AsyncAction {
+  return (dispatch) => {
+    openUrl(proof.humanUrl)
```

Comment:
side note: we could have this emit an action and have a saga that just listens for `openUrl` actions and does the side effecty stuff! this would be pure :)

The future is bright 🌅